### DISCUSSION TITLE
Minder crashes

### DISCUSSION THREAD

**Discussion by MartineMin** [phase1geo/Minder#325] [Q&A]
I have installed Minder on ubuntu. Sometimes i can't open a file already saved. Minder crashes. and then i can't open no more file .minder. I don't understand why I can't open the file. I have no message and i don't know what to do? I have attached a pdf of the file that i can't open
[refelxions_projeet_tzcldB.pdf](https://github.com/phase1geo/Minder/files/6254753/refelxions_projeet_tzcldB.pdf)

**Comment by phase1geo** [comment]
Could you open an issue for this?  I would like to take a closer look when I get the chance.
